website/views/static/community-guidelines.jade

```diff
@@ -81,6 +81,8 @@ block content
         img(class='pull-left', src='/community-guidelines-images/moderators.png', alt='')
         ul(class='pull-left list-unstyled')
           li
+            strong Beffymaroo
```

Comment:
People's names should always be written exactly as the person themself has
chosen. For moderators, you can see their chosen names in the list at the
top of Tavern chat. So for beffymaroo, it must be written like that.